libs/community/langchain_community/tools/brave_search/tool.py

```diff
@@ -33,7 +58,27 @@ def from_api_key(
         Returns:
             A tool.
         """
-        wrapper = BraveSearchWrapper(api_key=api_key, search_kwargs=search_kwargs or {})
+        wrapper = BraveSearchWrapper(
+            api_key=SecretStr(api_key), search_kwargs=search_kwargs or {}
+        )
+        return cls(search_wrapper=wrapper, **kwargs)
+
+    @classmethod
+    def from_search_kwargs(cls, search_kwargs: dict, **kwargs: Any) -> BraveSearch:
+        """Create a tool from search kwargs,
+        using environment variable BRAVE_SEARCH_API_KEY for api key.
```

Comment:
Google doc-string style
```suggestion

        Uses the environment variable BRAVE_SEARCH_API_KEY for the api key.
```